hunts/src/AnswerCell.js

```diff
@@ -1,31 +1,103 @@
+import { faEdit, faTrashAlt } from "@fortawesome/free-regular-svg-icons";
+import { faPlus } from "@fortawesome/free-solid-svg-icons";
 import { showModal } from "./modalSlice";
 
 export default function AnswerCell({ row, value }) {
   const { id: huntId } = useSelector((state) => state.hunt);
   const dispatch = useDispatch();
-  return value ? (
-    <span className="text-monospace">{value}</span>
-  ) : (
-    <Button
-      variant="outline-primary"
-      onClick={() =>
-        dispatch(
-          showModal({
-            type: "SUBMIT_ANSWER",
-            props: {
-              huntId,
-              puzzleId: row.values.id,
-              puzzleName: row.values.name,
-            },
-          })
-        )
-      }
-    >
-      Submit Answer
-    </Button>
+  if (row.original.guesses === undefined || row.original.guesses.length == 0) {
+    return (
+      <Button
+        variant="outline-primary"
+        onClick={() =>
+          dispatch(
+            showModal({
+              type: "SUBMIT_ANSWER",
+              props: {
+                huntId,
+                puzzleId: row.values.id,
+                puzzleName: row.values.name,
+              },
+            })
+          )
+        }
+      >
+        Submit Answer
+      </Button>
+    );
+  }
+  return (
+    <>
+      {row.original.guesses.map(({ id, text }) => (
+        <>
+          <span className="text-monospace">{text}</span>
+          <span
+            style={{ cursor: "pointer" }}
+            onClick={() =>
+              dispatch(
+                showModal({
+                  type: "EDIT_ANSWER",
+                  props: {
+                    huntId,
+                    puzzleId: row.values.id,
+                    answerId: id,
+                    text,
+                  },
+                })
+              )
+            }
+          >
+            <Badge pill variant="light">
+              <FontAwesomeIcon icon={faEdit} />
+            </Badge>
+          </span>{" "}
+          <span
+            style={{ cursor: "pointer" }}
+            onClick={() =>
+              dispatch(
+                showModal({
+                  type: "DELETE_ANSWER",
+                  props: {
+                    huntId,
+                    puzzleId: row.values.id,
+                    answerId: id,
+                  },
+                })
+              )
+            }
+          >
+            <Badge pill variant="light">
+              <FontAwesomeIcon icon={faTrashAlt} />
+            </Badge>
+          </span>
+          <br />
+        </>
+      ))}
+      <span
+        style={{ cursor: "pointer" }}
+        onClick={() =>
+          dispatch(
+            showModal({
+              type: "SUBMIT_ANSWER",
+              props: {
+                huntId,
+                puzzleId: row.values.id,
+                puzzleName: row.values.name,
+              },
+            })
+          )
+        }
+      >
+        <Badge pill variant="light">
```

Comment:
It kinda bugs me a bit having this button always visible, it adds some clutter. Here's what last year's mystery hunt looks like:

![image](https://user-images.githubusercontent.com/3475509/103267851-5b784580-4967-11eb-83dc-1f99c1337eac.png)

All the buttons feel like they draw my eyes away and make the answers harder to read. OTOH we definitely need edit/delete answer buttons in some form.

Not sure what the best thing to do is (I'm certainly no designer lol). What do you think of having a special "MULTIPLE ANSWERS" tag and only assuming that the puzzle takes multiple answers if it has this tag? I think nearly every puzzle we do will only have one answer so I feel like we should optimize for that case.

Reply:
I agree.

IMO the best thing to do is to actually get rid of all the buttons (including edit/delete), and have the answer be a link to a modal that allows you to modify, delete, or add.

Same can be said for the name column IMO. If you agree with this, I'd prefer to do both in the same, separate PR.